source/renderer/app/api/nodes/types.js

```diff
@@ -51,36 +51,40 @@ export type GetNetworkStatusResponse = {
   }
 };
 
+export type GetLatestAppVersionResponse = {
+  latestAppVersion: ?string,
+};
+
 export type Platform = {
-  signature: string,
   hash: string,
-  URL: string,
-  version: string,
   SHA256: string,
+  signature: string,
+  version: string,
+  URL: string,
 };
 
 export type Platforms = {
   ['windows' | 'darwin' | 'linux']: Platform,
```

Comment:
When platform is windows json will return win32. @nikolaglumac can you improve that please?